token/token.go

```diff
@@ -50,3 +50,9 @@ func GenerateKeypair(filename string) (err error) {
 	err = ioutil.WriteFile(filename+".pub", pubKeyPEM, os.FileMode(0644))
 	return
 }
+
+func KeypairExists(filename string) bool {
+	_, err1 := ioutil.ReadFile(filename + ".priv")
```

Comment:
may be we should check for existence of file using os.Stat and then checking error with 
 os.IsNotExist https://golang.org/pkg/os/#IsNotExist

if we want to also check for read permissions we should explicitly check that and return more useful message to make reason of failure more obvious.

Reply:
If we get rid of the "generate if doesn't exist" case, we no longer need to do an existence check. Just try to use it.

Reply:
The files are small, and the checking code is small. Why make it more complicated? I have almost no golang experience and this I understand :)

Reply:
ok, I take it back, ioutil provides enough information in err if it fails to read the file

Reply:
though it will be nice to print the msg on debug level to help in debugging